pkg/lib/common.ts

```diff
@@ -26,32 +28,32 @@ export function in_array(array, val) {
     return false;
 }
 
-export function is_function(x) {
+export function is_function(x: unknown): x is (...args: unknown[]) => unknown {
     return typeof x === 'function';
 }
 
-export function is_object(x) {
+export function is_object(x: unknown): x is object {
     return x !== null && typeof x === 'object';
 }
 
-export function is_plain_object(x) {
+export function is_plain_object(x: unknown): boolean {
     return is_object(x) && Object.prototype.toString.call(x) === '[object Object]';
 }
 
 /* Also works for negative zero */
-export function is_negative(n) {
+export function is_negative(n: number): boolean {
     return ((n = +n) || 1 / n) < 0;
 }
```

Comment:
This function looks just horrible. This `1 / n` thing is really non-obvious. It has a single consumer in `SeriesGrid.move`, could we just move and inline it there?

Reply:
Sure.

Reply:
Done.